index.js

```diff
@@ -21,7 +21,42 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+    Recipe.insertMany(data)
+    .then((data)=>{
+      for(let i =0 ; i<data.length;i++){
+        console.log(data[i].title)
+      }
+
+    Recipe.findOneAndUpdate({title:"Rigatoni alla Genovese",},{duration:100,},{
+      new:true,
+    })
+    .then(()=>{
+      console.log("se actualizó")
+    })
+    .catch((e)=>{
+      console.log("hay un error",e)
+    })
+    
+    Recipe.deleteOne({title:"Carrot Cake"},{new:true})
+    .then(()=>{
+      console.log("se eliminó el titulo")
+    })
+
```

Comment:
Al final de nuestras iteraciones podemos concatenar las promesas para que se cierre la coneccion con la base de datos
```suggestion
.then(() => {
            const closePr = mongoose.connection.close();
            return closePr;
          })
          .then(() => {
            console.log("DATABASE CLOSED!");
          });
```